src/phaser-game/scenes/FortNerf.ts

```diff
@@ -109,128 +108,136 @@ class FortNerf extends Phaser.Scene {
     // still animation function
     const createStillAnimation = (
       keyFrame: string,
+      spriteKey: string,
       frameNumber: number,
     ) => {
       this.anims.create({
         key: keyFrame,
-        frames: [{ key: 'player', frame: frameNumber }],
+        frames: [{ key: spriteKey, frame: frameNumber }],
         frameRate: 20,
       });
     };
+    // player animations
     // movement animations
-    createMoveAnimations('left', 3, 5);
-    createMoveAnimations('right', 6, 8);
-    createMoveAnimations('down', 0, 2);
-    createMoveAnimations('up', 9, 11);
+    createMoveAnimations('left', 'player', 3, 5);
+    createMoveAnimations('right', 'player', 6, 8);
+    createMoveAnimations('down', 'player', 0, 2);
+    createMoveAnimations('up', 'player', 9, 11);
 
     // still animations
-    createStillAnimation('leftstill', 4);
-    createStillAnimation('rightstill', 7);
-    createStillAnimation('downstill', 1);
-    createStillAnimation('upstill', 10);
+    createStillAnimation('leftstill', 'player', 4);
+    createStillAnimation('rightstill', 'player', 7);
+    createStillAnimation('downstill', 'player', 1);
+    createStillAnimation('upstill', 'player', 10);
 
-    // bullet animations
+    // other player animations
+    // movement animations
+    createMoveAnimations('leftTwo', 'otherPlayer', 3, 5);
+    createMoveAnimations('rightTwo', 'otherPlayer', 6, 8);
+    createMoveAnimations('downTwo', 'otherPlayer', 0, 2);
+    createMoveAnimations('upTwo', 'otherPlayer', 9, 11);
+
+    // still animations
+    createStillAnimation('leftstillTwo', 'otherPlayer', 4);
+    createStillAnimation('rightstillTwo', 'otherPlayer', 7);
+    createStillAnimation('downstillTwo', 'otherPlayer', 1);
+    createStillAnimation('upstillTwo', 'otherPlayer', 10);
 
-    // collision
-    const playerHit = () => {
-      player.setVelocity(0, 0);
-    };
-    this.physics.add.collider(
-      player,
-      walls,
-      playerHit,
-      undefined,
-      this,
-    );
     player.setCollideWorldBounds(true);
     // keyboard methods
-    upCursor = this.input.keyboard.addKey('UP');
-    leftCursor = this.input.keyboard.addKey('LEFT');
-    rightCursor = this.input.keyboard.addKey('RIGHT');
-    downCursor = this.input.keyboard.addKey('DOWN');
-    spaceBar = this.input.keyboard.addKey('SPACE');
-    // console.log(cursors);
-    this.physics.world.on('collide', (bullet: any, wall: any) => {
-      bullet.destroy();
+    this.input.keyboard.on('keydown', (e: { code: any }) => {
+      if (!pressedKeys.includes(e.code)) {
+        pressedKeys.push(e.code);
+      }
+    });
+    this.input.keyboard.on('keyup', (e: { code: any }) => {
+      pressedKeys = pressedKeys.filter((key) => key !== e.code);
+    });
+
```

Comment:
Sometimes you're using newlines between code blocks and other times not... whatever your choice, just try to be consistent.